eigenda/init.go

```diff
@@ -4,11 +4,24 @@ import (
 	"bytes"
 
 	"github.com/ethereum/go-ethereum/accounts/abi"
+	gethcommon "github.com/ethereum/go-ethereum/common"
 )
 
 var certDecodeABI abi.ABI
 
+// HistoricalEigenDAWasmRoots is a mapping of historical consensus artifacts (i.e, prover machine and binary)
+// that use EigenDAPreimageType=3
+var HistoricalEigenDAWasmRoots map[gethcommon.Hash]interface{}
+
 func init() {
+	HistoricalEigenDAWasmRoots =
+		map[gethcommon.Hash]interface{}{
+			gethcommon.HexToHash("0x2c9a9d645ae56304c483709fc710a58a0935ed43893179fe4b275e1400503ea7"): nil, // consensus-eigenda-v40
+			gethcommon.HexToHash("0x39a7b951167ada11dc7c81f1707fb06e6710ca8b915b2f49e03c130bf7cd53b1"): nil, // consensus-eigenda-v32.3
+			gethcommon.HexToHash("0xc723bd1be9fc564796bd8ce5c158c8b2f55d34afb38303a9fb6a8f0fda376edb"): nil, // consensus-eigenda-v32.2
+			gethcommon.HexToHash("0x04a297cdd13254c4c6c26388915d416286daf22f3a20e3ebee10400a3129dd17"): nil, // consensus-eigenda-v32.1
+		}
```

Comment:
nit: these are all hardcoded, no need to do this in init, just do it above where HistoricalEigenDAWasmRoots is defined.